scripts/update_version_for_prerelease.py

```diff
@@ -24,12 +24,17 @@
         prerelease_prefix = "b"
     elif prerelease_type == "rc":
         prerelease_prefix = "rc"
+    else:
+        raise RuntimeError(f"Unknown prerelease type: {prerelease_type}")
 
     current_version = get_current_package_version()
 
     # We can only transform a stable release version (X.Y.Z) to a prerelease version (X.Y.ZxxxN)
     if not re.match(r"^\d+\.\d+\.\d+$", current_version):
-        raise RuntimeError(f"The current version {current_version} does not match the proper semver format for stable releases (X.Y.Z)")
+        raise RuntimeError(
+            f"The current version {current_version} does not match the proper semver format for "
```

Comment:
```suggestion
            f"The current version {current_version} does not match the proper server format for "
```

Reply:
> 

Here, the semver stands for semantic version, so I'm skipping this suggestion